### PR TITLE
[TW review] Merchant Portal > Products

### PR DESCRIPTION
**Pull request by gibmirmilch**
## PR Description

https://spryker.atlassian.net/browse/AC-1232

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/creating-marketplace-concrete-product.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  You can remove a concrete product from the preview list by clicking the **Remove** icon.
+
+Once the product is created, it needs to be [activated by the Marketplace administrator in the Back Office](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/catalog/products/managing-products/managing-products.html#activating-a-product). Only the active marketplace products are displayed in the Merchant Portal and Marketplace Storefront.
+
+### Reference information: Create Concrete Products for [Abstract product name SKU]
+
+This page contains a drop-down menu that displays the super attribute values based on the super attribute selected while [creating a marketplace abstract product](/docs/marketplace/user/merchant-portal-user-guides/{{ page.version }}/products/abstract-products/creating-marketplace-abstract-product.html). When you select a product attribute value, a concrete product based on this value is displayed. In the *Concrete Products’ Preview* pane you can view the products to be created.
+
+By selecting **Autogenerate SKU**s, the SKU numbers for the concrete products are generated automatically, based on the SKU of their abstract product.
```
Comment (AdvaBD) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/creating-marketplace-concrete-product.md:44]: SKU prefix

**Review by AdvaBD — CHANGES_REQUESTED** [review]
can we arrange the topic of manage abstract products based on the order they are displayed in the drawer
all comments regarding abstract attributes images prices etc are relevant for the concrete product as well

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+---
+title: Creating a marketplace abstract product
+last_updated: Aug 11, 2021
+description: This document describes how to create marketplace abstract products in the Merchant Portal.
+template: back-office-user-guide-template
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md:5]: We don't have a separate template for Merchant Portal guides. We use a single back-office-user-guide-template. If needed, I can create a separate one for Merchant Portal guides. However, customers on the website will not be able to see it on the website, it's needed for the internal use.

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+1. On the *Products* page, click **Create Product**. *Create Abstract Product* drawer opens.
+
+2. Define the following settings:
+
+   1. Enter an **SKU Prefix**.
+
+   2. Enter a **Name** for the default locale. The rest of the locales will be defined once the product is created.
+
+   3. Select **Abstract product has 1 concrete product** if you are creating an abstract product with a single concrete product. *Create an Abstract Product with 1 Concrete Product* drawer opens.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md:28]: added

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+   {% info_block warningBox "Warning" %}
+
+   It is not possible to save the abstract product unless it is accompanied by at least one concrete product.
+
+   {% endinfo_block %}
+
+   
+   Select **Abstract product has multiple concrete products** if the abstract product you are creating will contain multiple concrete products assigned to it.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md:37]: added

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+      ![img](https://spryker.s3.eu-central-1.amazonaws.com/docs/User+Guides/merchant+portal+user+guides/Products/create-abstract-product-with-one-variant-mp.gif)
+
+
+
+      If the **Abstract product has multiple concrete products** was chosen:</br>
+
+   </br>
+
+      1. Select a super attribute that define your concrete products.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md:60]: added

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+      2. In the field next to the super attribute you've selected, select one or more product attribute values. Upon adding the super attribute values, the preview of the concrete products will be displayed.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md:61]: added

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+To return to the previous step, click **Back**.
+
+Once the product is created, it needs to be [activated by the Marketplace administrator in the Back Office](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/catalog/products/managing-products/managing-products.html#activating-a-product). Only the active marketplace products are displayed in the Merchant Portal and Marketplace Storefront.
+
+### Reference information: Create abstract product
+
+| ATTRIBUTE             | DESCRIPTION       |
+| ----------------------------- | ------------------------------------ |
+| SKU prefix                                      | Unique abstract product identifier that is used to track unique information related to the product. |
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md:89]: Autogenerate SKUs is described later, in the reference information. No need to add it here.

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+| Name                                            | The name of the abstract product that is displayed for the product on the Storefront. |
+| Abstract product has 1 concrete product         | Select this option when you want your abstract product to have a single concrete product. |
+| Abstract product has multiple concrete products | Select this option when you want your abstract product to have multiple concrete products. |
+
+### Reference information: Create an Abstract Product with 1 Concrete Product
+
+| ATTRIBUTE            | DESCRIPTION             |
+| --------------------- | ------------------------------------ |
+| Concrete Product SKU     | Unique product identifier that is used to track unique information related to the product. |
+| Autogenerate SKU         | Select this checkmark if you want the SKU to be generated automatically. By default, -1 is added to the abstract product SKU. For example, |
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md:99]: added.

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+
+This document describes how to create a merchant abstract product.
+
+## Prerequisites
+
+To start working with merchant abstract products, go to **Merchant Portal > Products**.
+
+Review the reference information before you start, or just look up the necessary information as you go through the process.
+
+## Creating a marketplace abstract product
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/creating-marketplace-abstract-product.md:16]: we call the features the way they are named in the release app https://release.spryker.com/features/view/108

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/creating-marketplace-concrete-product.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+  {% endinfo_block %}
+
+  4. Click **Save**.
+
+  **Tips and Tricks** </br>
+
+  You can remove a concrete product from the preview list by clicking the **Remove** icon.
+
+Once the product is created, it needs to be [activated by the Marketplace administrator in the Back Office](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/catalog/products/managing-products/managing-products.html#activating-a-product). Only the active marketplace products are displayed in the Merchant Portal and Marketplace Storefront.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/creating-marketplace-concrete-product.md:38]: updated the info

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/managing-marketplace-abstract-product-prices.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
+---
+title: Managing marketplace abstract product prices
+last_updated: Aug 11, 2021
+description: This topic describes how to manage marketplace abstract product prices in the Merchant Portal.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/managing-marketplace-abstract-product-prices.md:4]: added volume prices information

**Review by AdvaBD — CHANGES_REQUESTED** [review]

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
+---
+title: Managing a marketplace concrete product
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:2]:
```suggestion
title: Managing marketplace concrete products
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+Select one or more sorting parameters from the drop-down menu on the *Concrete Products* page to sort the existing concrete products. Choose among the following sort criteria:
+- SKU
+- Name
+- Status
+- Valid from
+- Valid to
+
+By default, the table is sorted descendingly by SKU.
+
+## (De)Activating a concrete product
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:31]:
```suggestion
## Activating and deactivating a concrete product
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md (hunk 12 above)
Comment (lenadoc) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:2]: Please change the link accordingly as well

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+
+1. On the *Concrete Products* page, next to the concrete product you want to activate, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *Concrete Product SKU, Name* page.
+2. In the *Status* pane, select *Concrete Product is online* to make the concrete product online.
+3. Click **Save**.
+
+To deactivate the product, deselect *Concrete Product is online* to make the concrete product offline.
+
+## Editing marketplace concrete product details
+
+To edit the existing marketplace concrete product:
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:43]:
```suggestion
To edit an existing marketplace concrete product:
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+
+1. On the *Concrete Products* page, next to the concrete product you want to edit, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *Concrete Product SKU, Name* page.
+2. In the *Stock* pane, enter the quantity in the *Quantity* field.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:46]:
```suggestion
2. In the *Stock* pane, populate the *Quantity* field.
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+3. In the *Name* pane, edit *Name* for every locale.
+4. In the *Description* pane, edit *Description* for every locale.
+5. In the *Validity Dates & Time* pane, populate the *Valid from* and *Valid to* fields.
+6. In the *Searchability* pane, from the drop-down menu select the locale(s) where the product is searchable.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:50]:
```suggestion
6. In the *Searchability* pane, from the drop-down menu select the locales where the product is searchable.
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 17]
```diff
@@ -0,0 +1,77 @@
...
+7. Click **Save**.
+
+### Reference information: *Concrete Product SKU, Name* page
+
+| PANE    | ATTRIBUTE     | DESCRIPTION | REQUIRED? |
+| -------------- | ---------------- | ----------- | --------- |
+| Status         |                  | Defines the status of the concrete product. |           |
+|                |  Concrete Product is online    | The check-box that once selected will make the product active and available in the store. |               |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:58]:
```suggestion
|                |  Concrete Product is online    | The selected check box makes the product active and available in store. |               |
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+| Stock          |                  | Defines the stock of the concrete product. |           |
+|                 |   Reserved Stock           | The number of items of this concrete product that are reserved according to *Orders*. |               |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:60]:
```suggestion
|                 |   Reserved Stock           | Number of items of this concrete product that are reserved according to *Orders*. |               |
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 19]
```diff
@@ -0,0 +1,77 @@
...
+|                  |  Quantity                 | The number of items available in the warehouse. The default is 0. | ✓ |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:61]:
```suggestion
|                  |  Quantity                 | Number of items available in the warehouse. The default is 0. | ✓ |
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 20]
```diff
@@ -0,0 +1,77 @@
...
+|                  |  Always in Stock           | The check-box that once selected will make the product always available to be purchased. |               |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:62]:
```suggestion
|                  |  Always in Stock           | The selected check box makes the product always available for purchase. |               |
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 21]
```diff
@@ -0,0 +1,77 @@
...
+| Name                  |  | The name of your product that will be displayed in the online store for your customers. | ✓ |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:63]:
```suggestion
| Name                  |  | Name of your product displayed on the Storefront. | ✓ |
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 22]
```diff
@@ -0,0 +1,77 @@
...
+|               |  Use Abstract Product name for all locales   | Once the checkbox is selected, the name from the abstract product is taken over. |              |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:64]:
```suggestion
|               |  Use Abstract Product name for all locales   | Select the check box to take over the name of the abstract. |              |
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 23]
```diff
@@ -0,0 +1,77 @@
...
+| Description           |  | The description of the product that your customer sees in the online store. |           |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:65]:
```suggestion
| Description           |  | Product description. |           |
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 24]
```diff
@@ -0,0 +1,77 @@
...
+|              |  Use Abstract Product description for all locales    | Once the checkbox is selected, the description from the abstract product is taken over. |       |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:66]:
```suggestion
|              |  Use Abstract Product description for all locales    | Select the check box to take over the description of the abstract.. |       |
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 25]
```diff
@@ -0,0 +1,77 @@
...
+| Validity Dates & Time |  | Defines the period of time when the product is in active state. The **Valid from** date triggers the activation, while the **Valid to** date triggers the deactivation. Either no dates can be selected, or both. |           |
+| Price                 |  | In this pane, you can manage prices for your concrete product. See [Managing marketplace concrete product prices](/docs/marketplace/user/merchant-portal-user-guides/{{ page.version }}/products/concrete-products/managing marketplace-concrete-product-prices.html) for more details. |           |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:68]: The link does not display as link on a website locally

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 26]
```diff
@@ -0,0 +1,77 @@
...
+| Images                |  | In this pane, you can manage image sets for your concrete product. See [Managing concrete product image sets](/docs/marketplace/user/merchant-portal-user-guides/{{ page.version }}/products/concrete-products/managing-marketplace-concrete-products-image-sets.html) for more details. |           |
+| Attributes            |  | In this pane, you can manage attributes for your product. See [Managing concrete product attributes](/docs/marketplace/user/merchant-portal-user-guides/{{ page.version }}/products/concrete-products/managing-marketplace-concrete-product-attributes.html) for more details. |           |
+| Searchability         |  | Defines the stores where the concrete product can be searched via the Search function in the online store. If not selected, no values will be displayed when searching for this product. |           |
+
+## Next steps
+
+- [Manage concrete product prices](/docs/marketplace/user/merchant-portal-user-guides/{{ page.version }}/products/concrete-products/managing marketplace-concrete-product-prices.html)
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-product.md:75]: the link does not work

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md [hunk 27]
```diff
@@ -0,0 +1,64 @@
...
+
+Review the reference information before you start, or just look up the necessary information as you go through the process.
+
+##  Adding images for marketplace concrete products
+
+To add an image for a marketplace concrete product:
+
+1. On the *Concrete Products* page, next to the concrete product you want to edit, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *Concrete Product SKU, Name* page. Scroll down to the *Images* pane.
+2. Click **+Add Image Set***. The empty cells appear.
+3. In the *SET NAME*, enter the name of the set.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md:22]:
```suggestion
3. In *SET NAME*, enter the name of the set.
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md [hunk 28]
```diff
@@ -0,0 +1,64 @@
...
+4. In the *IMAGE ORDERED*,enter the number which defines the order of the images displayed.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md:23]:
```suggestion
4. In *IMAGE ORDER*, enter the number which defines the order of the images displayed.
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md [hunk 29]
```diff
@@ -0,0 +1,64 @@
...
+5. In the *SMALL IMAGE URL*, enter the URL.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md:24]:
```suggestion
5. In *SMALL IMAGE URL*, enter the URL.
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md [hunk 30]
```diff
@@ -0,0 +1,64 @@
...
+6. In the *LARGE IMAGE URL*, enter the URL.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md:25]:
```suggestion
6. In *LARGE IMAGE URL*, enter the URL.
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md [hunk 31]
```diff
@@ -0,0 +1,64 @@
...
+
+## Editing images for marketplace concrete products
+
+To edit an image of the marketplace concrete product:
+
+1. On the *Concrete Products* page, next to the concrete product you want to edit, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *Concrete Product SKU, Name* page. Scroll down to the *Images* pane.
+2. Select a default or a specific locale you want to update images for.
+3. Update the following:
+    - SET NAME
+    - IMAGE ORDERED
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md:36]:
```suggestion
    - IMAGE ORDER
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md [hunk 32]
```diff
@@ -0,0 +1,64 @@
...
+    - SMALL IMAGE URL
+    - LARGE IMAGE URL
+4. Click **Save**.
+
+## Deleting images for marketplace concrete products
+
+To delete an image in the image set for the marketplace concrete product:
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md:43]:
```suggestion
To delete an image in the image set for a marketplace concrete product:
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md [hunk 33]
```diff
@@ -0,0 +1,64 @@
...
+
+1. On the *Concrete Products* page, next to the concrete product you want to edit, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *Concrete Product SKU, Name* page. Scroll down to the *Images* pane.
+2. Select a default or a specific locale you want to delete an image for.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md:46]:
```suggestion
2. Select a default or a specific locale you want to delete the image for.
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/managing-marketplace-abstract-product-image-sets.md [hunk 34]
```diff
@@ -0,0 +1,65 @@
...
+    - SMALL IMAGE URL
+    - LARGE IMAGE URL
+4. Click **Save**.
+
+## Deleting images for marketplace abstract products
+
+To delete an image in the image set for the marketplace abstract product:
+
+1. Next to the abstract product, which image you want to delete, hover over the three dots, or just click the line. This takes you to the *[Product name]*, *Abstract Product Details* tab. Scroll down to the *Images* pane.
+2. Select a default or specific locale you want to delete an image for.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/managing-marketplace-abstract-product-image-sets.md:46]:
```suggestion
2. Select a default or specific locale you want to delete the image for.
```

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/managing-marketplace-abstract-product-image-sets.md [hunk 35]
```diff
@@ -0,0 +1,65 @@
...
+3. Click the **Delete** icon.
+
+To delete an image set with its images for the abstract product, click **Delete Image Set**.
+
+## Reference information: Images pane
+
+| ATTRIBUTE       | DESCRIPTION                                                  |
+| --------------- | ------------------------------------------------------------ |
+| Set name        | Name of your image set.                                      |
+| Image ordered   | If you add several images to an image set, specify the order in which they are to be shown in the front end and back end using *Image ordered* field. The order of images is defined by the order of entered numbers where the image set with "0" is the first to be shown. |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/abstract-products/managing-marketplace-abstract-product-image-sets.md:56]: What are the "front end and back end" - Storefront and Back Office?

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md [hunk 36]
```diff
@@ -0,0 +1,64 @@
...
+3. Click the **Delete** icon.
+
+To delete an image set with its images for the concrete product, click **Delete Image Set**.
+
+### Reference information: Images pane
+
+| ATTRIBUTE       | DESCRIPTION                                                  |
+| --------------- | ------------------------------------------------------------ |
+| Set name        | Name of your image set.                                      |
+| Image ordered   | If you add several images to an image set, specify the order in which they are to be shown in the front end and back end using *Image ordered* field. The order of images is defined by the order of entered numbers where the image set with "0" is the first to be shown. |
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md:56]: What are the "front end and back end" - Storefront and Back Office?

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md [hunk 37]
```diff
@@ -0,0 +1,64 @@
...
+| Small image URL | Link of the image that is going to be used in the product catalogs. |
+| Large image URL | Link to the image that is going to be used on the product details page. |
+
+## Next steps
+
+- [Manage a concrete product](/docs/marketplace/user/merchant-portal-user-guides/{{ page.version }}/products/concrete-products/creating-marketplace-concrete-product.html)
+- [Manage concrete product prices](docs/marketplace/user/merchant-portal-user-guides/{{ page.version }}/products/concrete-products/managing marketplace-concrete-product-prices.html)
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202108.0/products/concrete-products/managing-marketplace-concrete-products-image-sets.md:63]: broken link

**Review by lenadoc — CHANGES_REQUESTED** [review]

**Review by lenadoc — APPROVED** [review]

**Pull request merged**